### DISCUSSION TITLE
Accepting Group Assignment Fails

### DISCUSSION THREAD

**Discussion by pbui** [community/community#63206] [GitHub Education]
### Hi everyone,

I have create a group assignment and decided to test out accepting the assignment by creating a group.

When I follow the link to accept the group assignment, I am taken to a page that says the following:

> Before you can accept this assignment, you must create or join a team. Be sure to select the correct team as you won’t be able to change this later.

When I enter in new team name and press the green "+ Create team" button, I get a red alert that says the following:

> GitHub team could not be created. There seems to be a problem on github.com, please try again.

I have tried multiple times and it behaves the same.

In the past, this has normally worked (I've done this for about 4-5 years), so I'm not sure if this is a temporary issue or a change from GitHub.

**Comment by MireilleBF** [comment]
have exactly the same message today. I also used to teach with GitHub, and I never encountered such a problem. I don't know what to do...

**Comment by erialc-cal** [comment]
Having the same issue today.

**Comment by momaarr27** [comment]
Same issue, anyone find any solutions?

**Comment by joaocaram** [comment]
Same issue here today, in a assignment which already had some groups, but now isn't accepting new groups.

**Comment by BorislavY** [comment]
Same issue here. The group assignment allowed for 10 teams to be created, and is no longer allowing the creation of new teams, giving the message "GitHub team could not be created. There seems to be a problem on github.com, please try again.". Has there been a solution for this yet?

**Reply by BorislavY** [reply]
Just an update to this. We were able to get around this error by just changing the team name. I am not sure why the name would have this effect, as there were no teams with the original name in the assignment, but hopefully this helps track the issue down.

**Reply by samus-aran** [reply]
Hi @BorislavY 👋 

Just to confirm that since you changed a specific team name that you were able to create it and then continue to create additional teams? Have you tried creating a team with the "problematic" name since?

**Reply by Tobeyforce** [reply]
A student of mine just encountered the same bug when clicking on "Create Team" after choosing name. When entering the group name, they were able to work around it by changing the group name, but clearly the error handling here from github needs some fixing :)

**Comment by jennirinker** [comment]
For anyone else who wanders across this thread: I think the issue is due to same team names across assignments. I made a thread here for that discussion: https://github.com/orgs/community/discussions/155223
